docs/developer.md

```diff
@@ -11,3 +11,23 @@
 * WindowMaker plugin is recommended
 * Adherence to the [Google Java Style](https://google.github.io/styleguide/javaguide.html) is mandatory
     * An IntelliJ Idea autoformatter XML configuration file is included in the repo for convenience
+
+
+## Log
+
+Logging in the plugin is handled by `log4j` from Apache. The default loglevel is `ERROR`, and the following loglevels are used, in increasing severity order:
+
+ * `TRACE`
+ * `DEBUG`
+ * `INFO`
+ * `WARN`
+ * `ERROR`
+ * `FATAL`
+
+The log configuration file's location is `eclipse-plugin/eclipse/cc.codechecker.eclipse.plugin/log4j.properties` before compilation.
+
+After compiling the plugin, you may change the log level (which specifies which *least severe* category will be logged) by opening the built `cc.codechecker.eclipse.plugin-0.0.1-SNAPSHOT.jar` file (with `mc` or any `zip` editor) and changing the `log4j.properties` file therein.
+
+The loglevel is specified as follows:
+
+    log4j.appender.log.threshold=ERROR
```

Comment:
please add:

After modifying the jar file, you will need to restart eclipse for the changes to take effect.